### ISSUE TITLE
How does the original data change into the processed data？

### ISSUE THREAD

**Issue by rookieLiu2018** [DJjjjhao/FIRA-ICSE#2]
I noticed that the data of difftoken.json and diffmark.json appears to have been processed. How is done?

**Comment by abhi-11nav** [comment]
Yeah, I've been trying to find the code for initial processing too and couldn't ? It would be really helpful if you could provide some more information on that ?

Thank you.

**Comment by Yuuoniy** [comment]
I wonder too, any updates?

**Comment by DJjjjhao** [comment]
We adopt the difftoken.json and diffmark.json from the benchmark processed by this paper "[Commit Message Generation for Source Code Changes](https://www.ijcai.org/proceedings/2019/0552.pdf)" directly, and the preprocess script can be found in their repository https://github.com/SoftWiser-group/CoDiSum/blob/master/data_process_tools.py. "Difftoken.json" is the tokens of the code, which can be obtained by parsing the code. "Diffmark.json" is the edit type of each token, and the tokens from the same line has the same mark ("-"/"+"/" ").

**Comment by vanessailana** [comment]
Do you use difftext.json, from CoDiSum?
